lua/entities/gmod_wire_holoemitter.lua

```diff
@@ -114,54 +118,66 @@ if CLIENT then
 			end
 		end
 
+		local points = self.Points
 		local selfpos = ent:GetPos()
 
-		local n = #self.Points
+		local ent_ltw = ent.LocalToWorld
 
-		if (n == 0 or self:GetNWBool("Active",true) == false) then return end
+		if (#self.Points == 0 or self:GetNWBool("Active",true) == false) then return end
 
-		for k=1, n do
-			local v = self.Points[k]
+		render_SetMaterial( matbeam )
+		for k, v in ipairs(points) do
```

Comment:
without luajit, a normal for loop is faster than ipairs. with luajit enabled, they are the same speed, so while in the worst case, this change actually decreases performance, most likely it won't affect it either way

Reply:
ipairs is actually faster with luajit

Reply:
I have seen benchmarks that claim otherwise so it probably depends on the version of luajit you are using. I doubt gmod has the latest version